public/coad_cptac_2019/meta_study.txt

```diff
@@ -0,0 +1,7 @@
+type_of_cancer: Colon Adenocarinoma
```

Comment:
@lizabethkatsnelson type_of_cancer in portal follows the Oncotree Codes. For this can you change it to coad. This will partially pass the validation that is currently failing.

Reply:
I will change it now, thanks!